### ISSUE TITLE
Show the tutorial on the rootNavigator

### ISSUE THREAD

**Issue by gianmarcocalbi** [RafaelBarbosatec/tutorial_coach_mark#93]
Hello!

In my app, where I have a nested navigator, if I `show` the tutorial, the overlay is pushed on such navigator while I would like to have it shown on the root navigator to cover the whole screen.

The `show` function of the tutorial_coach_mark.dart is
```dart
  void show() {
    Future.delayed(Duration.zero, () {
      if (_overlayEntry == null) {
        _overlayEntry = _buildOverlay();
        Overlay.of(_context)?.insert(_overlayEntry!);
      }
    });
  }
```

in order to accomplish what I am looking for, it may be changed to 
```dart
  void show({bool rootOverlay = false}) {
    Future.delayed(Duration.zero, () {
      if (_overlayEntry == null) {
        _overlayEntry = _buildOverlay();
        Overlay.of(_context, rootOverlay: rootOverlay)?.insert(_overlayEntry!);
      }
    });
  }
```

What do you think?
Thank you

**Comment by RafaelBarbosatec** [comment]
Good ideia!
I will add this suggestion!

**Comment by RafaelBarbosatec** [comment]
Available in 1.2.0 version.
